litmus-portal/frontend/src/components/BinarySwitch/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { Grid, Switch, Typography } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import React from 'react';
+
+const AntSwitch = withStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: 28,
+      height: 16,
+      padding: 0,
+      display: 'flex',
+    },
+    switchBase: {
+      padding: 2,
+      color: theme.palette.grey[500],
+      '&$checked': {
+        transform: 'translateX(12px)',
+        color: theme.palette.common.white,
+        '& + $track': {
+          opacity: 1,
+          backgroundColor: theme.palette.secondary.dark,
+          borderColor: theme.palette.secondary.dark,
+        },
+      },
+    },
+    thumb: {
+      width: 12,
+      height: 12,
+      boxShadow: 'none',
+    },
+    track: {
+      border: `1px solid ${theme.palette.grey[500]}`,
+      borderRadius: 16 / 2,
+      opacity: 1,
+      backgroundColor: theme.palette.common.white,
+    },
+    checked: {},
+  })
+)(Switch);
```

Comment:
Can we add a separate 'styles' file for this and proper units to height, width, margin, etc?

Reply:
Separate styles won't work here as this is a modified one...you can check the mui docs but yes the height and width I'll change it slipped my mind

Reply:
Okay, didn't know that. Thanks!